### ISSUE TITLE
v2.5.0 brеaks babel helpers (SyntaxError: Unexpected token typeof)

### ISSUE THREAD

**Issue by efender** [rollup/rollup-plugin-babel#55]
v2.4.0 result:
```js
var babelHelpers = {};
    babelHelpers.typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
...
```
v2.5.0 result:
```js
var typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
```
But `var __typeof` instead of `var typeof` it is ok:
```js
var __typeof  = typeof Symbol  ...
```

**Comment by Rich-Harris** [comment]
Are you able to create a small reproduction of the error? It's working as expected for me if I transpile a simple module like `console.log(typeof foo)` – the helper is being assigned to `var _typeof`. Have added a PR with a test that passes – #56

**Comment by efender** [comment]
I get this error for [lodash](https://github.com/lodash/lodash/tree/es) import (only v2.5.0, not v2.4.0):
```js
import { default as _template } from './libs/lodash-es/forEach';
console.log( typeof 1 );
```

**Comment by Rich-Harris** [comment]
So, it turns out that 2.5.0 didn't *introduce* a bug, but it did *expose* an existing bug in Rollup itself. Just released Rollup 0.27.1 which fixes the issue – if you install that, everything should work correctly.

**Comment by efender** [comment]
Thanks! It works!
